tests/meta-mender-ci/recipes-testing/mender-mock-server/files/mender-mock-server.py

```diff
@@ -28,9 +29,18 @@
 
 
 class SimpleHTTPRequestHandler(BaseHTTPRequestHandler):
+
+    log = logging.getLogger(__name__)
+
     def do_GET(self):
         self.read_body()
 
+        if re.match(
+            "/api/devices/v2/deployments/device/deployments/([a-f0-9-]*)/update_control_map",
```

Comment:
Uhm, what about [this](https://github.com/mendersoftware/meta-mender/blob/ca2dd9b3062d2406a1159142e2f9e8f75f9bfb1f/tests/meta-mender-ci/recipes-testing/mender-mock-server/files/mender-mock-server.py#L55-L72)? Were we wrong, and it's already there?